core/Controller/LoginController.php

```diff
@@ -336,9 +336,20 @@ public function tryLogin(Chain $loginChain,
 			);
 		}
 
+		$user = trim($user);
+
+		if (strlen($user) > 255) {
```

Comment:
We limit to 255 bytes or chars?
If the later:
```suggestion
		if (mb_strlen($user) > 255) {
```

Reply:
@Altahrim see : https://github.com/nextcloud/server/pull/44218#discussion_r1532363550